### ISSUE TITLE
A minor redis issue with Openwrt/Lede build (Mips32 arch)

### ISSUE THREAD

**Issue by st1905** [seclab-ucr/INTANG#40]
Hello,

I have successfully built the mips32 binary for the Lede 17.01.4 but when I fire it up, it just hangs there with a redis error at the log.

What might be the reason for this ?

[INTANG_0-1_mips_mips32.zip](https://github.com/seclab-ucr/INTANG/files/2499148/INTANG_0-1_mips_mips32.zip)
[intangd.zip](https://github.com/seclab-ucr/INTANG/files/2499151/intangd.zip)
![intang1](https://user-images.githubusercontent.com/4885199/47267441-8c20f100-d54c-11e8-9ec8-2ced7991fd27.png)
![intang2](https://user-images.githubusercontent.com/4885199/47267444-9511c280-d54c-11e8-859a-9ce52e3f37ff.png)

**Comment by st1905** [comment]
OK, I think I have now fixed this one got another issue, this time with iptables

![intangd3](https://user-images.githubusercontent.com/4885199/47267632-3d288b00-d54f-11e8-8f16-1d09759cd27c.png)

**Comment by st1905** [comment]
I got the iptables error fixed too, now it just says

1540130356.088580 [INFO] Building sync connection with redis server.
1540130356.092326 [INFO] Sync connection built successfully.
1540130356.097557 [INFO] Connecting to TCP DNS server.

And stops there.
![intangd4](https://user-images.githubusercontent.com/4885199/47267961-3d2a8a00-d553-11e8-8d6a-37ae773bc320.png)

**Comment by st1905** [comment]
INTANGD daemon exits after that last line. Redis server still runs

**Comment by st1905** [comment]
Can someone help me for the iptables rules (For Openwrt) ? I have no problem running it on my router but cant figure out the proper rules.

**Comment by LGA1150** [comment]
You may want to try my kernel-space implementation
https://github.com/LGA1150/netfilter-spooftcp

**Comment by st1905** [comment]
Thank you, i`m going to study this. 

This page was very useful > https://github.com/LGA1150/netfilter-spooftcp/wiki/Usage-Examples

**Comment by linwhitehat** [comment]
> Thank you, i`m going to study this.
> 
> This page was very useful > https://github.com/LGA1150/netfilter-spooftcp/wiki/Usage-Examples

I‘m using INTANG to test if I can visit sites blocked by GF, but I can not successfully, could you share the whole steps about using INTANG and setting the strategy?

**Comment by st1905** [comment]
> > Thank you, i`m going to study this.
> > This page was very useful > https://github.com/LGA1150/netfilter-spooftcp/wiki/Usage-Examples
> 
> I‘m using INTANG to test if I can visit sites blocked by GF, but I can not successfully, could you share the whole steps about using INTANG and setting the strategy?

I no longer use INTANG but at the time i was using it, the strategy i used was the 5. Netfilter-spooftcp is much easier to use and it works at the router level so one does`t need to apply any rules/software to the connected devices. My router now opens all restricted sites, you just need to compile a kernel module and an iptables extension. If you struggle with the iptables rules afterwards, i can send the ones i use here.
